library/percona

```diff
@@ -1,20 +1,18 @@
-# this file is generated via https://github.com/docker-library/percona/blob/2bc351a6c89c243851efc828d7ec19a21663df80/generate-stackbrew-library.sh
+Maintainers: Evgeniy Patlan <evgeniy.patlan@percona.com> (@EvgeniyPatlan),
+             Mykola Marzhan <mykola.marzhan@percona.com> (@delgod)
+GitRepo: https://github.com/percona/percona-docker.git
 
-Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
-             Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
-GitRepo: https://github.com/docker-library/percona.git
-
-Tags: 5.7.23-stretch, 5.7-stretch, 5-stretch, stretch, 5.7.23, 5.7, 5, latest
+Tags: 8.0.23-centos, 8.0-centos, 8-centos, 8.0.23, 8.0, 8
 Architectures: amd64
-GitCommit: 0f4b22adef232551c763308367d3b54d586a4a38
-Directory: 5.7
+GitCommit: f176d725d49b6c241263ea796eddb58f17e415f2
```

Comment:
> Just for comparison, the old percona:5.7-stretch is ~403MB, where the new percona:5.7-centos is ~577MB (and percona:8.0-centos is all the way up at ~721MB ❗️).

Yes, we know and fully understand this. Anyway, we believe that we need to use centos image as base, because it can simplify some 3rd party certifications for us.

> It would be preferred if this were a full fingerprint instead of just a long ID

fixed, please review

> It's probably also worth noting that the key appears to be dsa1024

change main repo sign key not a simple thing, but we understand security reason very well and thinking about using a stronger key.

> If /var/lib/mysql is created during the yum install

Fixed, please review

> That's a pretty long time to keep trying to connect -- does mysqld really take that long to initialize on an empty database for a first run sometimes? 😕

Fixed, please review

> but I wanted to mention it in case you get users complaining about it later to hopefully save you a little digging / hair-pulling.

Yes, I fully understand why it is needed, but we don’t want to run entrypoint.sh from root privileges, it is a hard decision for us, but we believe that we should have root privileges or sudo inside image, even if it is less convenient for end users, we prefer to be more secure.